<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFrozenSetDialog</class>
 <widget class="QDialog" name="CFrozenSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>固定行列</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>行数:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineeditRowNum</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineeditRowNum"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>列数:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineeditColumnNum</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="lineeditColumnNum"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>线型:</string>
     </property>
     <property name="buddy">
      <cstring>comboboxStyle</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QComboBox" name="comboboxStyle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>线颜色:</string>
     </property>
     <property name="buddy">
      <cstring>toolbuttonColor</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QToolButton" name="toolbuttonColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::DownArrow</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>线宽:</string>
     </property>
     <property name="buddy">
      <cstring>lineeditWidth</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QLineEdit" name="lineeditWidth">
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="pushbuttonCancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushbuttonOk">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushbuttonOk</sender>
   <signal>clicked()</signal>
   <receiver>CFrozenSetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushbuttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>CFrozenSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
